src/player.js

```diff
@@ -1104,6 +1139,9 @@ export default class Player extends FakeEventTarget {
       this._firstPlay = false;
       this.dispatchEvent(new FakeEvent(CustomEvents.FIRST_PLAY));
       this._posterManager.hide();
+      if (typeof this._playbackAttributesState.RATE === 'number') {
+        this.playbackRate = this._playbackAttributesState.RATE;
```

Comment:
place the update for playbackRate in ratechange event listener (need to add if we don't have this event)

Reply:
@dan-ziv 
in `ratechange` i'm saving the rate value. here i'm using it.

Reply:
So why here?
We cannot assign it also in `_handlePlaybackOptions`?

Reply:
no. you can change the rate only after loading/play